worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -66,7 +66,7 @@ private[celeborn] class Worker(
     workerArgs.host,
     workerArgs.port,
     conf,
-    Math.max(64, Runtime.getRuntime.availableProcessors()))
+    Math.min(64, Runtime.getRuntime.availableProcessors()))
```

Comment:
This change might introduce unexpected behavior. For example, if a worker runs in a single-core pod. This will hang workers which processing time-consuming RPC.

Reply:
modified